Rakefile

```diff
@@ -33,6 +33,15 @@ task release_all: [:webpack] do
   end
 end
 
+desc "Makes sure all official locales are complete and clean."
+task :check_locale_completeness do
+  DECIDIM_GEMS.each do |gem_name|
+    Dir.chdir("#{File.dirname(__FILE__)}/decidim-#{gem_name}") do
+      system({ "ENFORCED_LOCALES" => "en,ca,es" }, "rspec spec/i18n_spec.rb")
```

Comment:
This is the task supposed to be run from decidim's root. In that case, I wanted to hardcode it to just check the "officially supported" locales. I could make it configurable but I'm not sure if it's worth the effort...

Not that if the task is run from an app, the app locales are used instead, not via this task, but via https://github.com/decidim/decidim/pull/1330/files#diff-c3eb346890858ca91acb2f677044cdd1.